Makefile

```diff
@@ -137,4 +148,4 @@ push: container
 
 .PHONY: clean
 clean:
-	@-rm -vrf ${OUTPUT_DIR}
+	@$(RM) -vr ${OUTPUT_DIR}
```

Comment:
RM is one of Makefile environments. https://www.gnu.org/software/make/manual/html_node/Implicit-Variables.html

Reply:
Code files should be have a new line. More info: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
It's ok to use `@$(RM) -vr ${OUTPUT_DIR}` or `@-rm -vrf ${OUTPUT_DIR}`